### PR TITLE
fix duplicate alerts

### PR DESCRIPTION
**Pull request by DNRRomero**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py [hunk 1]
```diff
@@ -22,7 +22,6 @@
 FetchIncidentsStorage = TypedDict("FetchIncidentsStorage", {
     "last_fetched": str,
     "last_offset": str,
-    "first_run_at": str,
     "last_modified_fetched": str,
     "last_modified_offset": str,
     "zf-ids": list[int],
```
Comment (figarrido) [inline Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py:27]: Hay que cambiar esto también a `list[str]`

File: Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py [hunk 2]
```diff
@@ -1222,53 +1208,31 @@ def fetch_incidents(
...
-
-    next_run: FetchIncidentsStorage = {
-        "last_fetched": last_fetched_str,
-        "last_offset": last_offset_str,
-        "first_run_at": first_run_at.strftime(DATE_FORMAT),
-        "last_modified_fetched": last_modified_fetched_str,
-        "last_modified_offset": last_modified_offset_str,
-        "zf-ids": zf_ids,
-    }
+    zf_ids: set[str] = set(last_run.get("zf-ids", []))
```
Comment (figarrido) [inline Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py:1211]: Hay que castear los valores almacenados a str para hacer el cambio retrocompatible.

**Review by figarrido — CHANGES_REQUESTED** [review]
- [ ] Revisar si los tests y el comando `demisto-sdk pre-commit -i Packs/ZeroFox` siguen pasando.

File: Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py (hunk 2 above)
Comment (DNRRomero) [reply Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py:1211]: claro, para lo que ya existe almacenado

**Pull request merged**